server/routerlicious/packages/protocol-definitions/src/config.ts

```diff
@@ -18,12 +18,41 @@ export interface ISummaryConfiguration {
     maxAckWaitTime: number;
 }
 
+// Deli lambda configuration
+export interface IDeliServerConfiguration {
+    // Expire clients after this amount of inactivity
+    clientTimeout: number;
+
+    // Timeout for sending no-ops to trigger inactivity checker
+    activityTimeout: number;
+
+    // Timeout for sending consolidated no-ops
+    noOpConsolidationTimeout: number;
+}
+
+// Scribe lambda configuration
+export interface IScribeServerConfiguration {
+    // Enables generating service summaries
+    generateServiceSummary: boolean;
+
+    // Enables clearing the checkpoint cache after a service summary is created
+    clearCacheAfterServiceSummary: boolean;
+
+    // Enables writing a summary nack when an exception occurs during summary creation
+    ignoreStorageException: boolean;
+}
+
 /**
- * Key value store of service configuration properties provided as part of connection
+ * Key value store of service configuration properties
  */
-export interface IServiceConfiguration {
+export interface IServiceConfiguration extends IClientConfiguration, IServerConfiguration {
```

Comment:
Client needs to change from receiving IServiceConfiguration to IClientConfiguration only.  Can you create an issue to track that work to make sure it gets done?